docs/sliceutils/flatMap.md

```diff
@@ -0,0 +1,25 @@
+# FlatMap
+
+`func FlatMap[T any, R any](slice []T, mapper func(value T, index int, slice []T) []R) []R`
+
+FlatMap receives a slice of type T, executes the passed in slice-mapper function for each element in the slice, and
+returns a flattened slice containing all the elements from all the mapped slices.
+The function is passed the current element, the current index and the slice itself as function arguments.
+
+This is equivalent to Java streams' flatMap method / C#'s LINQ's SelectMany method.
```

Comment:
This part can me omitted. Many ML language family langs have this. E.g. contemporary ecmascript. 


```suggestion
```